tests/test_api.py

```diff
@@ -25,6 +25,12 @@ def test_bad_verb(self):
 
         self.assertIn("Invalid verb: BAD", str(e.exception))
 
+    def test_approve_job(self):
+        self.get_mock("approve_job_response")
+        resp = js(self.c.approve_job("workflow_id", "approval_request_id"))
+
+        self.assertEqual(resp["message"], "approved")
```

Comment:
did you mean `Accepted` (from the mock response) instead of `approved`?

Reply:
this results in tests failing. see https://travis-ci.org/github/alpinweis/pycircleci

Reply:
oops, I forgot to update the test after I checked the real response from CircleCI. :see_no_evil: 
I just fixed it. =)